scripts/update-statuses.ts

```diff
@@ -22,6 +23,24 @@ async function main() {
     accountId: MASTODON_ACCOUNT_ID,
   });
   const { data, account } = response;
+
+  // Check if the first status ID already exists in the current README
+  if (data.length > 0) {
```

Comment:
Remove the check for data.length. `getStatuses` will only return if statuses exist.